Source/Core/Common/Hash.h

```diff
@@ -4,14 +4,28 @@
 #pragma once
 
 #include <cstddef>
+#include <functional>
+#include <string_view>
 
 #include "Common/CommonTypes.h"
 
+namespace DiscIO
```

Comment:
Common shouldn't be dependent on DiscIO

Reply:
MD5.cpp already was in common and depended on DiscIO.

Reply:
The only function in MD5.cpp seems to have NetPlayClient.cpp as its only caller. I would suggest moving it either to there or to DiscIO.